src/scss/generated/_measures.scss

```diff
@@ -0,0 +1 @@
+// Generated Measures from Supernova. Do not edit manually.
```

Comment:
We need to get here all these tokens inherited from Spirit too because there should be only one source of truth for design tokens and that is Supernova.